.github/actions/install-chrome/action.yml

```diff
@@ -0,0 +1,16 @@
+name: Install Chrome browser and driver for Testing
+description: Download and install the latest available Chrome for Testing and Chromedriver
+
+runs:
+  using: composite
+  steps:
+    - id: install-chrome
+      name: Install Chrome
+      shell: bash
+      run: |
+        sudo apt-get remove google-chrome-stable
+        npx @puppeteer/browsers install chrome@stable
+        npx @puppeteer/browsers install chromedriver@stable
+        sudo ln -s /home/runner/work/keycloak/keycloak/chrome/**/chrome-linux64/chrome /usr/bin/chrome
+        sudo mv chromedriver/**/chromedriver-linux64/chromedriver $CHROMEWEBDRIVER/chromedriver
+        rm -rf chromedriver/
```

Comment:
I don't think we should be removing this directory, or the `chrome` directory for that matter. In fact, I think we should be [`actions/cache`](https://github.com/actions/cache) here. This will allow the downloaded binaries to be restored on subsequent runs, so that we don't need to re-download them off the internet.